provider/oci/cache_test.go

```diff
@@ -17,10 +17,11 @@ limitations under the License.
 package oci
 
 import (
-	"github.com/oracle/oci-go-sdk/v65/dns"
-	"github.com/stretchr/testify/assert"
 	"testing"
 	"time"
+
+	"github.com/oracle/oci-go-sdk/v65/dns"
+	"github.com/stretchr/testify/assert"
```

Comment:
Please avoid unrelated changes. Consider filing a separate PR for linting instead